### PR TITLE
vagrant 1.8

### PR DESCRIPTION
**Pull request by mitchellh**
Blog post!

### CODE REVIEW

File: source/blog/2015-12-21-vagrant-1-8.html.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+## Future
+
+Vagrant 1.8 is our first major Vagrant release since releasing
+[Otto](https://ottoproject.io). There was a lot of concern around the
+future of Vagrant since announcing Otto as the "successor to Vagrant."
+Many of the features of Vagrant 1.8 are the result of Otto needing them
+to reach that goal. Vagrant 1.8 is an example of how Vagrant will
+continue to improve -- in major ways -- alongside Otto.
+
+Vagrant is reaching that point once again where the featureset and
```
Comment (slackpad) [inline source/blog/2015-12-21-vagrant-1-8.html.md]: "that point"..."to the point" is kind of weird.

File: source/blog/2015-12-21-vagrant-1-8.html.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+Vagrant now uses linked clones to import boxes for VirtualBox and
+VMware. Linked cloning is a feature supported by many hypervisors that
+dramatically speeds up imports by only creating a differential disk
+versus copying the entire disk image of the virtual machine.
+
+Linked cloning will happen automatically if your system supports it.
+For providers other than VirtualBox and VMware, they can be updated outside
+of Vagrant to support linked cloning automatically.
+
+You won't notice any difference other than faster `vagrant up` times.
```
Comment (cbednarski) [inline source/blog/2015-12-21-vagrant-1-8.html.md]: Also less disk space used, right?

File: source/blog/2015-12-21-vagrant-1-8.html.md (hunk 1 above)
Comment (cbednarski) [inline source/blog/2015-12-21-vagrant-1-8.html.md]: Could say "The Vagrant 1.* series is mature and while we'll continue to make polish improvements, we'll be saving other dramatic changes for a future 2.0 release sometime in 2016."

File: source/blog/2015-12-21-vagrant-1-8.html.md (hunk 2 above)
Comment (cbednarski) [inline source/blog/2015-12-21-vagrant-1-8.html.md]: > Linked clones make `vagrant up` faster and can also dramatically reduce the disk space required for multi-VM configurations.

File: source/blog/2015-12-21-vagrant-1-8.html.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+Today we announce the release of [Vagrant 1.8](https://vagrantup.com).
+Vagrant is a tool for building and managing development environments.
+
+Vagrant 1.8 introduces a number of major features and hundreds of improvements
+and bug fixes. The major features are outlined below and the full list of
+improvements can be found in the official
+[CHANGELOG](https://github.com/mitchellh/vagrant/blob/v1.8.0/CHANGELOG.md).
+
+You can [download Vagrant 1.8](https://www.vagrantup.com/downloads.html) from the
+project website. There are no major backwards incompatibilities so you should
```
Comment (burzinpatel) [inline source/blog/2015-12-21-vagrant-1-8.html.md]: backward incompatibilities

File: source/blog/2015-12-21-vagrant-1-8.html.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+Vagrant now uses linked clones to import boxes for VirtualBox and
+VMware. Linked cloning is a feature supported by many hypervisors that
+dramatically speeds up imports by only creating a differential disk
+versus copying the entire disk image of the virtual machine.
+
+Linked cloning will happen automatically if your system supports it.
+For providers other than VirtualBox and VMware, they can be updated outside
+of Vagrant to support linked cloning automatically.
+
```
Comment (burzinpatel) [inline source/blog/2015-12-21-vagrant-1-8.html.md]: Providers other than VirtualBox and VMware, can be updated outside of Vagrant to support linked cloning automatically.

File: source/blog/2015-12-21-vagrant-1-8.html.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+And more! See the
+[full CHANGELOG](https://github.com/mitchellh/vagrant/blob/v1.8.0/CHANGELOG.md)
+for more details.
+
+## Future
+
+Vagrant 1.8 is our first major Vagrant release since releasing
+[Otto](https://ottoproject.io). There was a lot of concern around the
+future of Vagrant since announcing Otto as the "successor to Vagrant."
```
Comment (burzinpatel) [inline source/blog/2015-12-21-vagrant-1-8.html.md]: "successor to Vagrant".

**Comment by burzinpatel** [comment]
:+1:

**Pull request merged**